src/NzbDrone.Core/Indexers/IndexerDefinition.cs

```diff
@@ -10,6 +10,7 @@ public class IndexerDefinition : ProviderDefinition
         public DownloadProtocol Protocol { get; set; }
         public bool SupportsRss { get; set; }
         public bool SupportsSearch { get; set; }
+        public int Priority { get; set; } = IndexerDefaults.PRIORITY;
```

Comment:
Set the priority directly here.
I don't know that it makes sense to set to the lowest by default.

Reply:
Well I was thinking in terms that if someone wants to use it in a existing environment, lets say they have 6. If they want to make 1 indexer higher priority you only need to change that one indexer. If you were to say set them all to 1, to make one indexer a priority you need to change the other 5 indexers to something higher than 1.